config/locales/views/admin/en.log_events.yml

```diff
@@ -21,6 +21,7 @@ en:
           review: Order reviewed
           problem_queue: Order added to problem queue %{cause}
           price_change: Order's price was manually change
+          resolve_from_problem_queue: Order remove from problem queue
```

Comment:
Let's shorten this call this "Problem order resolved".

While we're in here, can we change the `price_change` to "Order's price was manually changed" (add a "d" to the end)?